scripts/in_each_e2e.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
Is this just to be silly? It's still dead code right? :)

Reply:
I actually use this locally quite a bit with `bazel test ... && ./scripts/in_each_e2e.sh bazel test ...` but I can put in my. home folder if you think it is not useful